shell.nix

```diff
@@ -0,0 +1,32 @@
+with import <nixpkgs> {
+  config = {
+    packageOverrides = pkgs: {
+      jemalloc = pkgs.jemalloc.overrideAttrs (oldAttrs: rec {
```

Comment:
Might be better to pin exact version of nixpkgs instead of pinning per package.

Reply:
Unfortunately, in the older `nixpkgs` release which contained `jemalloc-4.5.0` there was no `ghc-8.2.2`, used in `lts-10.8`, which caused numerous other problems.